analyzers/its/SonarAnalyzer.Testing.ImportBefore.targets

```diff
@@ -43,6 +43,7 @@
     <PropertyGroup Condition="$(SonarAnalyzerRegressionTestRunning)">
       <!-- Set the path to the SARIF report -->
       <ErrorLog>$(MSBuildStartupDirectory)\output\$(PROJECT)\$(AssemblyName)-$(ProjectGuid).json</ErrorLog>
+      <RazorCompilationErrorLog>$(MSBuildStartupDirectory)\output\$(PROJECT)\$(AssemblyName)-$(ProjectGuid).Views.json</RazorCompilationErrorLog>
```

Comment:
Reason for this is explained here https://github.com/SonarSource/sonar-scanner-msbuild/issues/949